src/dev/impl/DevToys/Helpers/CertificateHelper.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace DevToys.Helpers
+{
+#nullable enable
+    internal static class CertificateHelper
+    {
+        private const string BeginCertificate = "BEGIN CERTIFICATE";
+        private const string BeginCertificateRequest = "BEGIN CERTIFICATE REQUEST";
+        internal const string IncorrectPassword = "The specified network password is not correct";
+        internal const string FormatNotSupported = "Certificate Request format not currently supported.";
+
+        /// <summary>
+        /// Returns the friendly formatted, decoded certificate details.
+        /// CSR format is currently not supported in UWP, but is in .NET 7 <see href="https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.certificaterequest.loadsigningrequest?view=net-7.0"/>
+        /// PEM format is limited by the current framework version <see href="https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509certificate2.createfrompemfile?view=net-7.0"/>
+        /// </summary>
+        /// <param name="input">Certificate data</param>
+        /// <param name="password">Password for protected private key.  This should only be relevant for decoding pfx format.</param>
+        /// <returns>Certificate details</returns>
+        internal static bool TryDecodeCertificate(string input, string? password, out string? decoded)
+        {
+            decoded = null;
+            string publicCert = input;
+            if (publicCert.Contains('-'))
+            {
+                string[] splitCert = input.Split('-', StringSplitOptions.RemoveEmptyEntries);
+                for (int i = 0; i < splitCert.Length; i++)
+                {
+                    if (splitCert[i] == BeginCertificate)
```

Comment:
Can we please use `string.Equal(splitCert[i], BeginCertificate, StringComparison.Ordinal)` or `string.Equal(splitCert[i], BeginCertificate, StringComparison.OrdinalIgnoreCase)` instead?